www/js/desktop_scripts.js

```diff
@@ -133,9 +136,12 @@ module.exports = {
                     fs.unlinkSync(dbCompressed);
                     // Replace current DB file with new version
                     fs.renameSync(newDBPath, dbPath);
+                    if (dbPlatform === 'realm') {
+                      fs.renameSync(newDBSchema, dbSchema);
+                    }
                     module.exports.initDB();
                     // Delete old DBs
-                    const oldDBs = ['data.db'];
+                    const oldDBs = ['data.db', 'sttmdesktop.realm', 'sttmdesktop.realm.lock'];
```

Comment:
should `sttmdesktop.realm.management` folder also be cleaned up?

Reply:
We can add it as a TODO. The current implementation doesn't remove directories.